src/pages/buttons/demo/demo.html

```diff
@@ -0,0 +1,55 @@
+<ion-header>
+  <ion-navbar>
+    <ion-buttons start>
+      <button ion-button solid>
+        Header Button
```

Comment:
Can this be called something shorter? Maybe have 2 of them aligned right and 1 aligned left to show off the default, solid, and outline buttons in a navbar:

```
<ion-header>
  <ion-navbar>
    <ion-buttons start>
      <button ion-button solid>
        Solid
      </button>
    </ion-buttons>
    <ion-title>Buttons</ion-title>
    <ion-buttons end>
      <button ion-button>
        Default
      </button>
      <button ion-button outline>
        Outline
      </button>
    </ion-buttons>
  </ion-navbar>
</ion-header>
```